### PR TITLE
Update receiver address

### PR DESCRIPTION
**Pull request by leozoi**
Adding new Fields (state_name & city_name) for paremeter **receiver_address**

### CODE REVIEW

File: guides/payments/web-checkout/personalization.en.md [hunk 1]
```diff
@@ -321,6 +321,8 @@ To summarize all of the above, the following JSON shows all the data that can be
 	"shipments": {
 		"receiver_address": {
 			"zip_code": "5700",
+			"state_name": "State",
```
Comment (mminestrelli) [inline guides/payments/web-checkout/personalization.en.md]: @leozoi esto es pref

File: guides/payments/web-checkout/personalization.es.md [hunk 2]
```diff
@@ -324,6 +324,8 @@ Para resumir todo lo anterior, a continuación se muestran todos los datos que s
 	"shipments": {
 		"receiver_address": {
 			"zip_code": "5700",
+			"state_name": "State",
```
Comment (mminestrelli) [inline guides/payments/web-checkout/personalization.es.md]: Esto es pref

File: guides/payments/web-checkout/personalization.pt.md [hunk 3]
```diff
@@ -322,6 +322,8 @@ Para resumir todo lo anterior, a continuación se muestran todos los datos que s
   "shipments": {
     "receiver_address": {
       "zip_code": "5700",
+      "state_name": "State",
```
Comment (mminestrelli) [inline guides/payments/web-checkout/personalization.pt.md]: Esto es pref

File: reference/preferences/resource.yaml [hunk 4]
```diff
@@ -285,6 +285,18 @@ properties:
               en: Zip code.
               es: Código postal.
               pt: Código postal.
+        - state_name:
```
Comment (mminestrelli) [inline reference/preferences/resource.yaml]: Esto es pref

File: guides/payments/web-checkout/personalization.es.md (hunk 2 above)
Comment (leozoi) [reply guides/payments/web-checkout/personalization.es.md]: Done

File: guides/payments/web-checkout/personalization.pt.md (hunk 3 above)
Comment (leozoi) [reply guides/payments/web-checkout/personalization.pt.md]: Done

**Pull request merged**